lib/adagrams.rb

```diff
@@ -0,0 +1,117 @@
+# chooses 10 letters randomly from the letter bag
+def draw_letters()
```

Comment:
You have included the same file twice, as `adagrams.rb` and `lib/adagrams.rb`. Probably only the `lib` one should be included here.